platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/EventIntake.java

```diff
@@ -147,14 +151,17 @@ public EventIntake(
      * @param event the event
      */
     public void addUnlinkedEvent(final GossipEvent event) {
```

Comment:
Is all usage of the phase timer on the same thread?  The same timer is used in 3 different methods.  `addUnlinkedEvent`, `addEvent` and `EventValidator.validateEvent()`.    These methods are sent through SwirldsPlatform into a variety of places and I am not confident they are all on the same thread.

Reply:
Yes, all usages are on the intake thread.

Reply:
The access from different threads occurs because PCES replay uses the same code pathways as the standard event intake thread. I talked this over with @cody-littley, and his opinion is that even if these metrics were to be skewed during PCES replay, it wouldn't be a big deal.

Also worth noting is that even though PCES replay and intake are on separate threads, they should not be operating simultaneously, since a node replaying events won't be gossiping.